modules/deploy/pages/redpanda/manual/resilience/shadowing.adoc

```diff
@@ -0,0 +1,602 @@
+* **Task status**: Health of replication tasks across brokers
+* **Lag information**: Replication lag per partition
+
+[[shadow-link-metrics]]
+=== Metrics
+
+Shadowing provides comprehensive metrics to track replication performance and health:
+
+* `redpanda_shadow_link_shadow_lag` - The lag of the shadow partition against the source partition, calculated as source partition LSO minus shadow partition HWM. Monitor this gauge metric by `shadow_link_name`, `topic`, and `partition` to understand replication lag for each partition.
+
+* `redpanda_shadow_link_total_bytes_fetched` - The total number of bytes fetched by a sharded replicator (bytes received by the client). This count metric is labeled by `shadow_link_name` and `shard` to track data transfer volume from the source cluster.
+
+* `redpanda_shadow_link_total_bytes_written` - The total number of bytes written by a sharded replicator (bytes written to the write_at_offset_stm). This count metric uses `shadow_link_name` and `shard` labels to monitor data written to the shadow cluster.
+
+* `redpanda_shadow_link_client_errors` - The number of errors seen by the client. Track this count metric by `shadow_link_name` and `shard` to identify connection or protocol issues between clusters.
+
+* `redpanda_shadow_link_shadow_topic_state` - Number of shadow topics in the respective states. This gauge metric is labeled by `shadow_link_name` and `state` to monitor topic state distribution across your shadow links.
+
+* `redpanda_shadow_link_total_records_fetched` - The total number of records fetched by the sharded replicator (records received by the client). Monitor this count metric by `shadow_link_name` and `shard` to track message throughput from the source.
+
+* `redpanda_shadow_link_total_records_written` - The total number of records written by a sharded replicator (records written to the write_at_offset_stm). This count metric uses `shadow_link_name` and `shard` labels to monitor message throughput to the shadow cluster.
+
+For complete metrics reference, see xref:reference:public-metrics-reference.adoc[].
+
+=== Monitoring best practices
+
+==== Health check procedures
+
+Establish regular monitoring workflows to ensure shadow link health:
+
+**Health checks:**
+[,bash]
+----
+# Check all shadow links are active
+rpk shadow list | grep -v "ACTIVE" || echo "All shadow links healthy"
+
+# Monitor lag for critical topics  
+rpk shadow status <disaster-recovery-link> | grep -E "LAG|Lag"
+----
+
+==== Alerting thresholds
+
+Configure monitoring alerts for:
+
+* **High replication lag**: When `redpanda_shadow_link_shadow_lag` exceeds your RPO requirements
+* **Connection errors**: When `redpanda_shadow_link_client_errors` increases rapidly
+* **Shadow link state changes**: When shadow links move from `ACTIVE` to `PAUSED`
+* **Topic state changes**: When topics move to `FAULTED` state
+* **Throughput drops**: When bytes/records fetched drops significantly
```

Comment:
Make sure to add Next steps at the end. You may want to link in this file to the [pre-disaster checklist](https://deploy-preview-1381--redpanda-docs-preview.netlify.app/current/deploy/redpanda/manual/resilience/emergency-shadowing/#pre-disaster-checklist)